src/view/dataElements/xdmObject.jsx

```diff
@@ -57,101 +69,210 @@ const XdmObject = ({
+        </Alert>
+      )}
+      {schemasMetaStatus === STATUS_LOADED && schemaStatus === STATUS_LOADED && (
+        <div className="u-flex u-gapTop u-minHeight0">
+          <div className="XdmObject-treeContainer u-flexShrink0 u-overflowXAuto u-overflowYAuto">
+            <InfoTipLayout tip="Here you can use the form to explore and edit the xdm object of the selected schema.">
+              <FieldLabel labelFor="xdmTree" label="XDM Object" />
+            </InfoTipLayout>
+            <XdmTree
+              formikProps={formikProps}
+              selectedNodeId={selectedNodeId}
+              onSelect={setSelectedNodeId}
+            />
+          </div>
+          <div className="u-gapLeft2x">
+            <div>
+              {selectedNodeId ? (
+                <NodeEdit
+                  formState={formState}
+                  onNodeSelect={setSelectedNodeId}
+                  selectedNodeId={selectedNodeId}
+                />
+              ) : (
+                <NoSelectedNodeView
+                  sandboxMeta={selectedSandboxMeta}
+                  schemaMeta={selectedSchemaMeta}
+                  previouslySavedSchemaInfo={
+                    initInfo.settings && initInfo.settings.schema
+                  }
+                />
+              )}
+            </div>
+          </div>
+        </div>
+      )}
     </div>
   );
 };
 
 XdmObject.propTypes = {
   initInfo: PropTypes.object.isRequired,
   formikProps: PropTypes.object.isRequired,
+  sandboxesMeta: PropTypes.array,
+  selectedSandboxMeta: PropTypes.object,
+  setSelectedSandboxMeta: PropTypes.func,
   schemasMeta: PropTypes.array,
+  schemasMetaStatus: PropTypes.string,
+  setSchemasMetaStatus: PropTypes.func,
   selectedSchemaMeta: PropTypes.object,
-  setSelectedSchemaMeta: PropTypes.func
+  setSelectedSchemaMeta: PropTypes.func,
+  schemaStatus: PropTypes.string
 };
 
 const XdmExtensionView = () => {
+  const [sandboxesMeta, setSandboxesMeta] = useState();
   const [schemasMeta, setSchemasMeta] = useState();
+  const [schemasMetaStatus, setSchemasMetaStatus] = useState();
+  const [selectedSandboxMeta, setSelectedSandboxMeta] = useState();
   const [selectedSchemaMeta, setSelectedSchemaMeta] = useState();
+  const [schemaStatus, setSchemaStatus] = useState();
 
   // TODO: break apart this extension view
   return (
     <ExtensionView
       getInitialValues={({ initInfo }) => {
-        return fetchSchemasMeta({
+        return fetchSandboxes({
           orgId: initInfo.company.orgId,
           imsAccess: initInfo.tokens.imsAccess
         })
-          .then(_schemasMeta => {
-            setSchemasMeta(_schemasMeta);
+          .then(_sandboxesMeta => {
+            let sandboxName = null;
+            // if sandboxes are returned from the api call
+            if (_sandboxesMeta.sandboxes && _sandboxesMeta.sandboxes.length) {
+              setSandboxesMeta(_sandboxesMeta);
+              // default to the first item in the list
```

Comment:
Should we default to the sandbox that has "isDefault: true"?

Reply:
Yes, I think so. I will make the change